Cargo.toml

```diff
@@ -369,6 +369,17 @@ tokio = { workspace = true, features = ["rt"] }
 
 [patch.crates-io]
 # This is a temporary patch to get `cargo vendor` to work with the `uv` and pep508_rs` crates.
+file_url = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_cache = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_conda_types = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_digest = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_lock = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_networking = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_repodata_gateway = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_shell = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_solve = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
+rattler_virtual_packages = { git = "https://github.com/conda/rattler", rev = "a7370b5a304c32883e7beefed43f810305008434" }
```

Comment:
I just released rattler, can you update this?

Reply:
<img width="927" alt="image" src="https://github.com/user-attachments/assets/d054ecf5-a2b0-4ad9-9034-8061fbed0a35" />
don't see an update yet

Reply:
https://github.com/conda/rattler/actions/runs/13010357407/job/36286509636#step:4:315

It failed on an LFS error.

Reply:
[`2591571` (#2909)](https://github.com/prefix-dev/pixi/pull/2909/commits/2591571081e5a16fcf4162380d17e48b7403ff50)